tests/integration/internal/stepfuncs/helm.go

```diff
@@ -24,6 +24,16 @@ const (
 	envNameHelmNoDependencyUpdate = "HELM_NO_DEPENDENCY_UPDATE"
 )
 
+// HelmVersion returns a features.Func that will run helm version
+func HelmVersionOpt() features.Func {
+	return func(ctx context.Context, t *testing.T, envConf *envconf.Config) context.Context {
+		_, err := helm.RunHelmCommandAndGetOutputE(t, ctxopts.HelmOptions(ctx), "version")
```

Comment:
Shouldn't we attach this to the context as well, if we're getting it?

Reply:
Will it help with anything?

Reply:
🤷 seems strange to call the command just to see the version in logs, but I guess it's fine.